README.md

```diff
@@ -64,44 +68,46 @@ Example:
 ## Key scenarios
 [//]: # ([If there are a suite of interacting APIs, show how they work together to solve the key scenarios described.])
 
-### 1. Transform-style 'detached' behaves like 'preserve-3d' on platforms which do not support page decomposition.
+### 1. Transform-style `detached` behaves like `preserve-3d` on platforms which do not support page decomposition.
 
-We don't want authors to design different CSS for immersive vs flat browsers. If a user agent renders on a flat surface, it can use the 'preserve-3d' value which will render the page visually the same.
+We don't want authors to design different CSS for immersive vs flat browsers.
+If a user agent renders on a flat surface, transform-style `detached` falls back to `preserve-3d` which will render the page visually the same.
 
 [//]: # ([Description of the end-user scenario])
 
 [//]: # (// Sample code demonstrating how to use these APIs to address that scenario.)
-### 2. When detached elements are nested, only the top most element will signal that its children at detached
 
-Allowing detached descendants of detached nodes significantly complicates the implementation and introduces multiple corner cases. We propose that descendents with `detached` are treated as `preserve-3d`
+### 2. Nodes with transform-style: `flat` prevent descendents from being detached.
 
-[//]: # (TODO: We need to explan this, but I cant think of the best explanation)
+Transform style `flat` and effects that cause flattening ([spec](https://drafts.csswg.org/css-transforms-2/#grouping-property-values)) will cause `detached` fall back to `preserve-3d` behavior in descendent elements.
 
+The one exception is root node which allows detached descendants even though an implicit `flat` transform-style is applied.
 
-### 3. Effects such as clip applied to parents of detached nodes are not applied to detached nodes.
+### 3. Only the top-most frame can use this feature.
 
-The detached elements and their descendants are rendered onto surfaces that are distinct from their parents. Any effect applied to parents will be applied during the compositing stage and will not be applied to the detached elements.
+iframes should not be allowed to created detached content.
```

Comment:
nit: we can do it: <iframe>, easier to read.

Reply:
I think we can leave it as-is. we don't decorate tags anywhere else in the explainer.